samcli/commands/pipeline/bootstrap/guided_context.py

```diff
@@ -40,6 +42,13 @@ def run(self) -> None:
                 type=click.STRING,
             )
 
+        if not self.region:
+            self.region = click.prompt(
+                "\nAWS region (the AWS region where the environment infrastructure resources will be deployed to)",
+                type=click.STRING,
+                default="us-east-1",
```

Comment:
should we default to `us-east-1` or boto.session.region?

Reply:
updated, good catch